webrender/src/prim_store.rs

```diff
@@ -347,6 +346,9 @@ pub struct PrimitiveStore {
     pub cpu_gradients: Vec<GradientPrimitiveCpu>,
     pub cpu_metadata: Vec<PrimitiveMetadata>,
     pub cpu_borders: Vec<BorderPrimitiveCpu>,
+    // the indices witihin gpu_data_32 of the image primitives to be resolved by
+    // the renderer instead of the render backend.
+    pub deferred_image_primitives: Vec<(ExternalImageKey, GpuStoreAddress)>,
```

Comment:
I don't think we need to have `primitives` in the name, since technically an external texture can be a part of anything, not necessarily it's own `Image` type primitive. Perhaps, `external_images`?